cdb2api/cdb2api.c

```diff
@@ -1127,7 +1127,7 @@ void cdb2_set_comdb2db_info(const char *cfg_info)
     cdb2cfg_override = 1;
     len = strlen(cfg_info) + 1;
     CDB2DBCONFIG_BUF = malloc(len);
-    strncpy(CDB2DBCONFIG_BUF, cfg_info, len);
+    strcpy(CDB2DBCONFIG_BUF, cfg_info);
```

Comment:
compiler is able to deduce that len = strlen(cfg_info) + 1; is same as what we use in strncpy, so throws warning. Strcpy is safe in this case since we malloced 1 more than len, and will strcpy at most len.